### PR TITLE
Refactor body parts hierarchy on 8.2.0 mediawiki

### LINKED ISSUE #164: [Human] body-part hierarchy
The Body-part hierarchy under the base schema appears to be predominantly concerned with human anatomy, although it is stated to be any part of an organism. Nevertheless,  the hierarchy could benefit from more detailed and structured categorization, especially for biomechanical studies and sensor placement. Below is a potential revision. Definitions of currently defined tags have been removed for abstraction, and new potential tags have been added in boldface (no tags are removed). This (or a similar hierarchy) will also help provide a pathway for a more precise annotation of surface anatomy, mainly aimed toward sensor placement, which @sjeung, @JuliusWelzel, and I are pursuing.

* Body-part
  * Head
    * Hair
    * Ear
    * Face
      * Cheek, Chin, Eye, Eyebrow, Forehead, Lip, Nose, Mouth, Teeth
  * **Neck** [The part of the body connecting the head to the torso, containing the cervical spine and vital pathways of nerves, blood vessels, and the airway.]
  * Torso
    * Torso-chest
    * Torso-back
    * **Abdomen** [The part of the body between the thorax and pelvis.]
      * Waist, Hip, Buttocks, **Genitalia**
  * Upper-extremity
    * Upper-arm
      * Shoulder
    * Forearm
      * Elbow
    * Hand
      * Wrist, Palm, Knuckles
      * Finger, Index-finger, Little-finger, Middle-finger, Ring-finger, Thumb
  * Lower-extremity
    * **Upper-leg** [Upper part of the leg between hip and knee.]
      * Thigh
    * **Lower-leg** [Part of the leg between the knee and ankle.]
      * Knee, Calf, Shin
    * Foot
      * Ankle, Heel, Instep, Toes, Big-toe, Little-toe
      
CC @dorahermes  and @monique2208 for comments, edits and suggestions.

### PR DESCRIPTION
**Pull request by neuromechanist**
Following the discussion in #164, these changes will add a couple of terms and lightly adjust the anatomical hierarchy.

This pull request will close #164.

### CODE REVIEW

File: standard_schema/hedwiki/HED8.2.0.mediawiki [hunk 1]
```diff
@@ -196,41 +196,47 @@ Each term in this vocabulary has a human-readable description and may include ad
...
-***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips.]</nowiki>
-***** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
 ***** Torso-chest <nowiki>[The anterior side of the thorax from the neck to the abdomen.]</nowiki>
-***** Gentalia <nowiki>{deprecatedFrom=8.1.0}[The external organs of reproduction.]</nowiki>
-***** Hip <nowiki>[The lateral prominence of the pelvis from the waist to the thigh.]</nowiki>
-***** Waist <nowiki>[The abdominal circumference at the navel.]</nowiki>
+***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips, often regarded as the back]</nowiki>
+***** Abdomen <nowiki>[The part of the body between the thorax and the pelvis.]</nowiki>
+***** Pelvis <nowiki>[The bony structure at the base of the spine supporting the legs.]</nowiki>
+****** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
```
Comment (VisLab) [inline standard_schema/hedwiki/HED8.2.0.mediawiki:206]: I don't think Buttocks should go under Pelvis. It should be at the same level.  It is possible that we need to do:

**** Torso  
**** Torso-part  
***** Buttocks  

Now that we are going to ontologies -- a strict is-a will be enforced.

File: standard_schema/hedwiki/HED8.2.0.mediawiki [hunk 2]
```diff
@@ -196,41 +196,47 @@ Each term in this vocabulary has a human-readable description and may include ad
...
 **** Upper-extremity <nowiki>[Refers to the whole superior limb (shoulder, arm, elbow, wrist, hand).]</nowiki>
-***** Elbow <nowiki>[A type of hinge joint located between the forearm and upper arm.]</nowiki>
+***** Upper-arm <nowiki>[Portion of arm between shoulder and elbow.]</nowiki>
+****** Shoulder <nowiki>[Joint attaching upper arm to trunk.]</nowiki>
 ***** Forearm <nowiki>[Lower part of the arm between the elbow and wrist.]</nowiki>
+****** Elbow <nowiki>[A type of hinge joint located between the forearm and upper arm.]</nowiki>
 ***** Hand <nowiki>[The distal portion of the upper extremity. It consists of the carpus, metacarpus, and digits.]</nowiki>
+***** Wrist <nowiki>[A joint between the distal end of the radius and the proximal row of carpal bones.]</nowiki>
+****** Palm <nowiki>[The part of the inner surface of the hand that extends from the wrist to the bases of the fingers.]</nowiki>
+****** Knuckles <nowiki>[A part of a finger at a joint where the bone is near the surface, especially where the finger joins the hand.]</nowiki>
```
Comment (VisLab) [inline standard_schema/hedwiki/HED8.2.0.mediawiki:219]: Same thing here with adding parts.  We need to discuss...

**Review by VisLab — CHANGES_REQUESTED** [review]
This reorganization has raised an interesting issue.  All of the items in this hierarchy are "Body-part".  We have both Body and Body-part at the top level -- but the whole-part relationship was not followed.  We should probably discuss.  With the ontology addition we can add part-of relationships, but the HED hierarchy should probably do a strict is-a.

File: standard_schema/hedwiki/HED8.2.0.mediawiki (hunk 2 above)
Comment (neuromechanist) [reply standard_schema/hedwiki/HED8.2.0.mediawiki:219]: Sorry, the Wrist should be under Hand.

File: standard_schema/hedwiki/HED8.2.0.mediawiki (hunk 1 above)
Comment (neuromechanist) [reply standard_schema/hedwiki/HED8.2.0.mediawiki:206]: Teeth is in the Mouth (cavity). The same way that is under the Head hierarchy?

**Comment by VisLab** [comment]
I think we have to modify the schema under `Body-part` to preserve the is-a relationship. This will involve adding additional `x-part` organizational tags.  Because annotators mainly look for tags not paths, I don't think this will be a burden. We can put additional links in the ontology representation and create alternative hierarchical paths.

@smakeig @monique2208 @neuromechanist @dorahermes @dungscout96  : As a reminder we already had this discussion in the transition from HED2 to HED3.  HED2 had a mixed hierarchy with `has-a` and `is-a` relationships intermixed.  We had a schema attribute `predicateType` to distinguish. It made things very difficult.  We agreed to go with strict `is-a` to clean up the schema and assure search generality.  We removed the `predicateType` schema attribute. I know this is annoying not to have the parts.....

`* predicateType <nowiki> [One of propertyOf, subclassOf, passThrough -- used to facilitate mapping to OWL or RDF.]</nowiki>`

One thing that we could do is to introduce an 'isPartOf' schema attribute that works like `suggestedTag`.  This would inherited and would alert users about the relationship. What do you think?  @smakeig this might partially address your suggestion.

**Comment by neuromechanist** [comment]
I'll close this PR and open another one based on the Pre-release Schema.

**Pull request closed without merging**